visualizer/filters-options.css

```diff
@@ -0,0 +1,63 @@
+.filters-options{
+  padding   : 10px 14px;
+  box-shadow: 3px 0px 4px -1px rgba(0,0,0,0.7) inset;
+  color     : var(--grey-highlight);
+}
+
+.filters-options .side-bar-close-btn {
+  position: absolute;
+  top: 1px;
+  right: 1px;
+}
+
+.filters-options ul {
+  margin    : 0;
+  padding   : 0;
+  list-style: none;
+}
+
+.filters-options ul ul {
+  padding-left: 24px;
+}
+
+.filters-options .options label {
+  align-items     : flex-start;
```

Comment:
There's some inconsistency here between rules and properties sometimes being laid out like this to match the property start line and them being free-flowing. Might be nice to stick to one, which in this case would just be without extra indentation to matching existing styles.

Reply:
Yes, 
I personally like this style as I find it easier to read (keys alphabetically sorted and values vertically aligned)
```css
.filters-options .options label {
  align-items     : flex-start;
  background-color: transparent;
  cursor          : pointer;
  display         : flex;
  line-height     : 1.3;
  padding         : 0.7em 5px;
  width           : 100%;
}
```
But, as you previously said, it would be nice if we eventually come up with an agreed set of formatting rules to be enforced via script (and supported by the IDE of choice)  ;)